### PR TITLE
add kpouget to OWNERS file

### PR DESCRIPTION
**Pull request by ArangoGutierrez**
Signed-off-by: Carlos Eduardo Arango Gutierrez <carangog@redhat.com>

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -8,3 +8,5 @@ reviewers:
   - Deepthidharwar
   - ArangoGutierrez
   - dagrayvid
+  - kpouget
+component: "Node Feature Discovery Operator"
```
Comment (zvonkok) [inline OWNERS:12]: What is this for?

Comment (zvonkok) [reply OWNERS:12]: The component?

Comment (ArangoGutierrez) [reply OWNERS:12]: a recommendation from the ART team

**Pull request merged**